bartfeenstra/betty
# Fix package metadata and data files

This fixes https://github.com/bartfeenstra/betty/issues/937
This fixes https://github.com/bartfeenstra/betty/issues/964

## How to test
- Run `pip install git+https://github.com/bartfeenstra/betty.git@fix-package-0.2`
- Confirm the installation succeeds
- Confirm `betty --version` outputs `Betty, version development`
